README.md

```diff
@@ -67,6 +67,16 @@ let player = Player(options: options)
 player.attachTo(yourView, controller: self)
 ```
 
+#### Media Control
+
+It controls all the user interaction between UI and the playback, it relies only on playback abstraction, therefore, this component should work in playback.
+
+It's a set of `Stack Views` divided into panels (`Top`, `Center` and `Bottom`) and can be used to organize the components on the screen.
+
+#### Media Control Plugins
+
+Allow providing controls for the `Player`. There are default plugins available with basic user interactions like: `Play/Pause`, `TimeIndicator`, `Seekbar` and `Fullscreen`.
```

Comment:
The `MediaControl` plugins can be used to control the playback but can be used to other approaches that doesn't involve that: displaying a video's title, for example.

What do you think about providing a better description? Like:

1) It should inherits from `MediaControlPlugin`
2) Should contain the `panel` and `position`
3) Should have a unique name

And the link to the examples (`PlayButton.swift`, `Seekbar.swift` and so on)

Reply:
Done.